src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaOverview/MediaOverview.js

```diff
@@ -9,6 +9,7 @@ import MediaCollection from '../../containers/MediaCollection';
 import CollectionStore from '../../stores/CollectionStore';
 import mediaOverviewStyles from './mediaOverview.scss';
 
+const MEDIA_ROUTE = 'sulu_media.detail';
```

Comment:
This should be the `sulu_media.detail.information_taxonomies` route, so that the content is already there after navigating.